src/ACEtk/ContinuousEnergyTable.hpp

```diff
@@ -48,6 +49,10 @@ class ContinuousEnergyTable : protected Table {
   block::DLW dlwp_;
   block::YP yp_;
 
+  block::NU dnu_;
+  block::BDD bdd_;
+  block::DLW dned_;
```

Comment:
[design question]
I know it probably doesn't matter, but all of these are available in all tables. This should only be available for projectiles that can cause fission? Or would the check be "checking for the sake of checking"?

Reply:
These have been replaced with std::optional versions later on. Relying on empty() was just too unreliable. That change also resolves the point you make.